### DISCUSSION TITLE
New label: Help Wanted

### DISCUSSION THREAD

**Discussion by NetSysFire** [CleverRaven/Cataclysm-DDA#62914] [Show and tell]
There are three kinds of feature requests I have seen over time while triaging issues in the repo:

1. The urgent-ish quality of life improvement, which is actually a bug report. E.g "Make \<thing\> much less tedious"
2. Mediumterm things such as #62909, which is targeted by this label.
3. Vague-ish complex longterm feature requests, which are handled by stalebot, e.g #62334

In this case, I would deem this label helpful because those mediumterm things are generally able to be done with the things we have implemented and would improve gameplay. This of course depends on the issue and the one labelling things but this is more of a general thing I have noticed.

I went ahead and made this label, so please feel free to discuss this here. I only tagged a few issues with this so far, so just in case my brain is broken again and this makes no sense at all, I did not waste too much time and there is not much to clean up.

**Comment by Zireael07** [comment]
Nitpick: "make much less tedious" - I think a word or even a couple of words got eaten?

**Reply by GuardianDll** [reply]
What exactly this label mean? "this issue is possible to do pretty easily"? We have good first issue for this. "I want to do it, but don't know how"? Then yes, i see potential in this - there was #58580, that died because author got no help, but i don't think the issues you mention are exact this

**Reply by NetSysFire** [reply]
@Zireael07 I wrote \<thing\> but did not escape the \<\>. Ugh. Thanks for spotting.

@GuardianDll This is a "This is not a priority but feel free to work on it, as it will improve the game in some way", primarily for issues. Maybe this or a different label can help there, but I imagined that the Help Wanted label is basically a "I'm bored and want to hack something" remedy. Tbh I would love some more detailed labels, such as splitting Crafting / Construction into Crafting and a seperate Construction label. Same for Terrain / Furniture / Traps.

I would not be opposed to rewrite the description of the label I just made so it can fit PRs, too.

**Reply by GuardianDll** [reply]
> This is not a priority but feel free to work on it, as it will improve the game in some way

Isn't that like the idea of feature request? If it's something we are highly opposed, it will be closed pretty fast, otherwise it will stay here untill someone resolve it ~~or stale bot close it as inactive~~

**Reply by NetSysFire** [reply]
Yes, but:
>There are three kinds of feature requests

See the top post. Encouraging contributors to work on it might just work.

**Comment by NetSysFire** [comment]
A somewhat welcome side effect is that Github will display "*N* issues need help", too.

**Reply by Maleclypse** [reply]
I think this is a great idea and we should go ahead and test it out.

**Reply by anothersimulacrum** [reply]
This label has already been applied to a number of issues.

**Reply by NetSysFire** [reply]
Yes, I slapped it on basically any reasonable feature requests or bug reports. Some time ago I also discussed what to name it and what to put as description in the IRC and someone brought up Debian's "Sooner if you help", which is pretty good, too.
